### PR TITLE
Add modules for producing line and page breaks

### PR DESCRIPTION
**Pull request by adelhult**
Resolves GH-162. Adds the `[newline]` and `[newpage]` modules in the layout package, I'm open to other name suggestions.

### CODE REVIEW

File: packages/layout/src/main.rs [hunk 1]
```diff
@@ -70,15 +81,54 @@ fn manifest() {
...
             eprintln!("Package does not support {other}");
         }
     }
 }
 
-fn transform_flex(from: &str, to: &str) {
+fn transform_newline(to: &str, input: Value) {
+    if let Value::String(data) = &input["data"] {
+        if !data.is_empty() {
+            eprintln!("The newline module was fed with some input. Maybe this was a misstake? Consider adding delimiters like this: [newline]().");
```
Comment (Forsinge) [inline packages/layout/src/main.rs:103]: Should probably be spelt "mistake"

**Review by Forsinge — APPROVED** [review]
Looks good!

This reminds me I forgot to add tests for row and center modules, I will open a new issue for this.

Commented on a rather insignificant typo, up to you if you want to fix it.

File: packages/layout/src/main.rs (hunk 1 above)
Comment (adelhult) [reply packages/layout/src/main.rs:103]: Haha, I'll fix the *mistake*!

Comment (Forsinge) [reply packages/layout/src/main.rs:103]: Good one :)

**Pull request merged**